### ISSUE TITLE
Subscribing to mutations

### ISSUE THREAD

**Issue by jjnesbitt** [paleo/direct-vuex#60]
I'm looking to subscribe to a state mutation like is done [here](https://vuex.vuejs.org/api/#subscribe).

I know this can be accomplished through `store.original.subscribe`, but that doesn't seem to preserve type information. Is there a supported way to do this in direct-vuex?

**Comment by paleo** [comment]
No… For the moment I don't intend to work on this feature. Sorry.
